stdlib/public/Platform/glibc.modulemap.gyb

```diff
@@ -320,6 +320,13 @@ module SwiftGlibc [system] {
       }
 % end
 
+% if CMAKE_SDK in ["CYGWIN"]:
```

Comment:
Foundation's `./Foundation/NSFileHandle.swift` uses Glibc's `L_SET`, `L_INCR`, `L_XTND`. In Linux, it is enough only include `unistd.h`, but Cygwin's `unistd.h` doesn't define them. 

To support L_xxx macro, Cygwin should include `sys/file.h` which defines them. But other platforms which already support Foundation would already have the definitions.